projects/klippa/ngx-enhancy-forms/src/lib/form/form-validation-error/form-validation-error.ts

```diff
@@ -0,0 +1,36 @@
+import { InjectionToken } from '@angular/core';
+
+export class FormValidationError extends Error {
+	_path: string;
+	_message: string;
+
+	constructor(path: string, message: string) {
+		super(message);
+		this.name = 'FORM_VALIDATION_ERROR';
+		this._path = path;
+		this._message = message;
+	}
+
+	get path() {
+		return this._path;
+	}
```

Comment:
for consistency's sake, `getPath()`

Reply:
I could also make it like this?
```
class ... {
  public readonly path: string;
  private message: string;
}
```

Reply:
actually, this would be sufficient:

```
export class FormValidationError extends Error {
	public readonly name = 'FormValidationError';
	public readonly path: string;

	constructor(path: string, message: string) {
		super(message);
		this.path = path;
	}
}
```

Reply:
I could also have name reference `this.constructor.name`. That should be static afaik.